django/cantusdb_project/main_app/templates/chant_edit.html

```diff
@@ -79,6 +79,28 @@
                         </div>
                     </div>
 
+                    {% if chant.volpiano %}
```

Comment:
As I understand it, you are moving this from below, right? So shouldn't there be an equivalent deletion starting at line 191?

Reply:
Nope, no duplication! I can see where the confusion arose - I copied this code (and the new code in chant_proofread.html) from the Chant Detail view. Up until now, NewCantus entirely lacked a text-and-melody preview on the Chant Edit and Chant Proofread pages - it was only present in the Chant Detail view. OldCantus, in contrast, had a preview near the bottom of its Chant Edit page. So now NewCantus's Chant Edit page has a text-and-melody preview, as it should, but it's in a more useful place on the page than it was in OldCantus's.